test/jquery.i18n.test.js

```diff
@@ -49,13 +49,16 @@
 			QUnit.start();
 			assert.strictEqual( i18n.locale, 'en', 'Locale is English' );
 			assert.strictEqual( $.i18n( 'message_1' ), 'ONE', 'Simple message' );
+			assert.strictEqual( $.i18n( 'message_4', 5 ), 'Total $1 of $1500', 'Parameter replacement escaped' );
 		} );
 		assert.strictEqual(
 			$.i18n( 'This message key does not exist' ),
 			'This message key does not exist',
 			'This message key does not exist'
 		);
 		assert.strictEqual( $.i18n( 'Hello $1', 'Bob' ), 'Hello Bob', 'Parameter replacement' );
+		assert.strictEqual(  $.i18n( 'Price of {{PLURAL:$1|an item|$1 items}} is $2, you get \\$2 discount', 5, 2000 ),
```

Comment:
Double space here, but better to have one parameter per line instead of mixing them up.